Resources/views/content/line.html.twig

```diff
@@ -0,0 +1,3 @@
+{% set content_name=ez_content_name(content) %}
+
+<p>{{ content_name }}</p>
```

Comment:
I don't know if it's better. But I know that if we don't need it, then we can remove it :)